app/data/Owners.scala

```diff
@@ -0,0 +1,37 @@
+package data
+
+import model._
+
+trait Owners {
+  def stacks: Set[(String, SSA)]
+
+  def all: Set[Owner] = stacks
+    .groupBy(_._1)
+    .map { case (ownerId, stacks) =>
+      Owner(ownerId, stacks.map(_._2))
```

Comment:
although I understand that the `stacks` here comes from the `case(ownerId, stacks)` maybe for the sake of clarity rename that, so it won't be confused with the outside stacks?